cli.sh

```diff
@@ -66,6 +66,10 @@ set_main_variables() {
     if [ "${CURRENT_GIT_DIR}" = "--git-common-dir" ]; then
         CURRENT_GIT_DIR=".git"
     fi
+
+    # Global IFS for loops
+    IFS_COMMA_NEWLINE="
```

Comment:
Oh, I missed this. You say `COMMA_NEWLINE` but doesn't actually include `,`

Reply:
damm, thx!

Reply:
At least we know that some test fail like that :)